src/secrets.js

```diff
@@ -12,6 +13,7 @@ export const MIXPANEL_PROJECT_TOKEN =
 
 export const WS_RELAY_URL = process.env.REACT_APP_WS_RELAY_URL;
 export const IS_BURN = process.env.REACT_APP_IS_BURN;
+export const SSO_PROVIDER_ID = process.env.REACT_APP_SSO_PROVIDER_ID;
```

Comment:
See earlier comments about configuring this via `venue` config rather than ENV.

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1546/commits/cd6ed2a394579a1a482db8a751630924672ad528